### PR TITLE
648 add samples for feature assertions of api fluent

### LINKED ISSUE #648: add samples for featureAssertions of api-fluent
## Code related feature
See for the initial request: https://github.com/robstoll/atrium/issues/624

With this issue we want to add samples for featureAssertions.kt of api-fluent

Following the things you need to do:
 
*api-fluent*
- [ ] create FeatureExtractorSamples in apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples
  - [ ] add one function per function in featureAssertions.kt (see AnyExpectationSamples)
  - [ ] refer to the sample in the KDoc of the corresponding function in featureAssertions via `@sample` (check out `toEqual` in anyExpectations.kt to see how you have to refer to the sample)

## Your first contribution?
- Write a comment `I'll work on this` if you would like to take this issue over. 
  This way we get the chance to revise the description in case things have changed in the meantime, we might give you additional hints and we can assign the task to you, so that others do not start as well.
- See [Your first code contribution](https://github.com/robstoll/atrium/blob/master/.github/CONTRIBUTING.md#your-first-code-contribution) for guidelines.  
- Do not hesitate to ask questions here or to contact us via [Atrium's slack channel](https://kotlinlang.slack.com/team/U3DE1TXKP) if you need help
  ([Invite yourself](https://slack.kotlinlang.org/) in case you do not have an account yet).

### PR DESCRIPTION
**Pull request by iljakorneckis**
Closes #648

I'm a little unsure about naming of the sample methods, so welcome any feedback and will make all required changes asap.
______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/main/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

**Comment by iljakorneckis** [comment]
Hey @robstoll, can I bug you for a review?

**Review by robstoll — CHANGES_REQUESTED** [review]
@iljakorneckis I am sorry, it looks like I forgot to press the submit button (I actually reviewed the same day you submitted :man_facepalming: )

**Comment by iljakorneckis** [comment]
All good, @robstoll I'll make the changes asap and re-submit

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/FeatureExtractorSamples.kt [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+    fun itsFeature() {
+        val person = Person(name = "John Smith", age = 25)
+
+        expect(person)
+            .its { age } // subject is now Int
+            .toEqual(25)
+
+        fails {
+            expect(person)
+                .its({ age }) { toEqual(30)} // fails, subject still Person afterwards
```
[the remark this answers is not in the document]
Comment (iljakorneckis) [reply apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/FeatureExtractorSamples.kt]: Oops! Fixed

**Comment by robstoll** [comment]
@iljakorneckis thanks for the changes and the contribution as such :tada:

**Comment by iljakorneckis** [comment]
Uh, @robstoll, I haven't done all the changes requested in the review. Sorry if I was unclear, when I resolved one of them. Should I open a separate PR for the rest?

**Comment by robstoll** [comment]
@iljakorneckis ah sorry, I figured you don't have time for the rest. I adjusted the last bits myself.

**Comment by robstoll** [comment]
@iljakorneckis If you should have more time available, then don't hesitate to take a look at other issues :slightly_smiling_face:

**Comment by iljakorneckis** [comment]
@robstoll Hah, thank you for tackling the rest! I will definitely look at other issues that I can take. Learned a lot about multi-platform Kotlin from this project and started using it myself. :+1:

**Comment by robstoll** [comment]
@iljakorneckis just be aware of that this project actually uses the old multi-platform plugins (we need to update to the new soonish). If you are interested about the new plugins then take a look at the following issue: #744

**Pull request merged**